kaznet/apps/main/tests/viewsets/test_submissions.py

```diff
@@ -661,7 +661,8 @@ def test_submission_time_sorting(self):
             response.data['results'][0]['submission_time'],
             Submission.objects.order_by(
```

Comment:
I don't understand why we need to manually remove microseconds from the Queryset.